src/dk/simongray/datalinguist/ml/crf.clj

```diff
@@ -101,4 +100,19 @@
   (let [word-col (first (ds/column-names feature-dataset))]
     (predict-crf feature-dataset word-col thawed-model)))
 
-(ml/define-model! :corenlp/crf train predict {})
+
+
+(def metamorph-ml-options
+  (map
+   (fn [[k v]]
+     (hash-map :name k :type :string :default v))
+   default-opts
+   ))
```

Comment:
What's with the format of the options here? I would expect a map, but this is a sequence of maps?

```clojure
({:default "true", :name :useClassFeature, :type :string}
 {:default "true", :name :useTypeSeqs, :type :string}
 {:default "true", :name :usePrevSequences, :type :string}
 {:default "true", :name :useDisjunctive, :type :string}
 {:default "true", :name :useWord, :type :string}
 {:default "ner-model.ser.gz", :name :serializeTo, :type :string}
 {:default "true", :name :useNext, :type :string}
 {:default "true", :name :useSequences, :type :string}
 {:default "true", :name :useTypeySequences, :type :string}
 {:default "true", :name :usePrev, :type :string}
 {:default "1", :name :maxLeft, :type :string}
 {:default "true", :name :useTypeSeqs2, :type :string}
 {:default "none", :name :wordShape, :type :string}
 {:default "true", :name :noMidNGrams, :type :string}
 {:default "true", :name :useNGrams, :type :string}
 {:default "6", :name :maxNGramLeng, :type :string})
```